src/resources/MachineLearning/DNEConfiguration/tests/DNEConfiguration.spec.ts

```diff
@@ -46,6 +46,16 @@ describe('DNEConfiguration', () => {
         });
     });
 
+    describe('parseDocumentExtractionQuery', () => {
+        it('should make a GET call to the specific DNEConfiguration url', () => {
+            const aString: string = 'lala';
```

Comment:
In the [TypeScript doc](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-annotations-on-variables) - I think it says it automatically infers the types in our code.. so no annotation would be here as the string 'lala' is inferred as a type `string`